### PR TITLE
Revised Tripp Lite model detection to work with SNMPWEBCARDs

### PR DESCRIPTION
**Pull request by psquiddy**
Removed code that seeks to remove "Tripp Lite" from model string; this is not present in all Tripp Lite devices.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by psquiddy** [comment]
Hardware detail on Mitel 3300 after upgrade to 9.x code on system shows "ppc"; would be more useful to include the "Cxi" and "MCD 9.0" information.

File: includes/polling/os/msl.inc.php [hunk 1]
```diff
@@ -18,7 +18,11 @@
 $oids             = array("mitelAppTblProductVersion.10.1.3.6.1.4.1.1027.1.6.1", "mitelAppTblProductDescr.10.1.3.6.1.4.1.1027.1.6.1");
 $mitelapptbl_data = snmp_get_multi_oid($device, $oids, "-OUQnt", "MITEL-APPCMN-MIB");
 
-$hardware = preg_replace('/;VerSw.*$/', '', (preg_replace('/^.*VerHw:/', '', $mslhw)));
```
Comment (murrant) [inline includes/polling/os/msl.inc.php:21]: Is this change supposed to be in here?

File: includes/polling/os/poweralert.inc.php [hunk 2]
```diff
@@ -1,13 +1,12 @@
 <?php
 
 // .1.3.6.1.2.1.33.1.1.2.0 = STRING: "TRIPP LITE PDUMH20HVATNET"
+// .1.3.6.1.2.1.33.1.1.2.0 = STRING: "SU1500XL"
 // .1.3.6.1.2.1.33.1.1.4.0 = STRING: "12.04.0052"
 // .1.3.6.1.2.1.33.1.1.5.0 = STRING: "sysname.company.com"
 // .1.3.6.1.4.1.850.100.1.1.4.0 = STRING: "9942AY0AC796000912"
 // .1.3.6.1.4.1.850.10.2.2.1.12.1 = STRING: "This Is My Location"
 $hardware    = snmp_get($device, 'upsIdentModel.0', '-Ovq', 'UPS-MIB');
-$hardware    = preg_split('/TRIPP\ LITE/', $hardware);
```
Comment (murrant) [inline includes/polling/os/poweralert.inc.php:9]: change the preg_split to an str_replace or a preg_match()...

**Comment by psquiddy** [comment]
Revised code per advice to use "str_replace"

Thanks,

Clay Berlo

**Comment by psquiddy** [comment]
Tried to put in conditional code to detect newer outputs similar to the way
they were previously processed.  After version 9.x, $hardware only outputs
CPU information, rather than platform-based info.  Previously, this
provided the type of Mitel device (i.e. CX-II, Mxe, etc.).  Current parsing
only outputs cpu (i.e. 'ppc').  The new code submitted returns the hardware
type and software level.

Thanks,

Clay Berlo

**Comment by Jellyfrog** [comment]
Please see failed test, https://travis-ci.com/github/librenms/librenms/jobs/377943751

**Comment by murrant** [comment]
Upstream detection changed, I don't think these are still valid, feel free to reopen if you update them.

**Pull request closed without merging**